CONTRIBUTING.md

```diff
@@ -205,6 +205,13 @@ There are two ways to run TensorFlow unit tests.
     ```bash
     bazel test ${flags} tensorflow/python/kernel_tests:softmax_op_test
     ```
+    
+    For a single/parametrized test e.g. `test_capture_variables` in `tensorflow/python/saved_model/load_test.py`:
+    
+    (Requires `python>=2.7`)
+    
+    ```bash
+    bazel test //tensorflow/python/saved_model:load_test --test_arg=-- --test_arg=-k=*LoadTest.test_capture_variables*  --test_sharding_strategy=disabled
```

Comment:
```suggestion
    bazel test //tensorflow/python/saved_model:load_test --test_arg=-- --test_arg=-k=*LoadTest.test_capture_variables*
```

It should work this way too, right? I'm reticent in adding flags that are not directly related to the documented feature as that would imply that the additional flags always need to be passed.

Reply:
The rationale of that flag in this specific section was: 
In a single internal test also if parametrized probably you don't want the have by default the sharding overhead.
I  this rationale is valid I suppose that in this context this is a quite safe default for the contributors.
An alternative if you prefer we could add an extra line with the same command + `--test_sharding_strategy=disabled` with a specific extra claim. In that case what claim would you like?

Reply:
P.s. Of course my rationale really depended on our AVG expected test complexity. See https://github.com/bazelbuild/bazel/issues/2113#issuecomment-264054799

Reply:
If instead we want to suggest a general "pattern" filtering in this section instead of the most common case of a pattern for a single test we could probably have something different and then probably we could directly remove the sharding flag and rename the section.

Reply:
I think it's better to mention the flag in a separate section